packages/webpack-plugin/src/index.js

```diff
@@ -26,6 +33,24 @@ class LoadablePlugin {
       errorDetails: false,
       timings: false,
     })
+
+    // include extra asset keys if present
+    stats.assets = stats.assets.map((asset) => {
```

Comment:
I think we will be fine without that isn't it?

Reply:
Actually, you're absolute right!